opensearch_dashboards.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "dashboardsMaps",
```

Comment:
According to example, they recommend "mapsDashboards" i believe: https://github.com/opensearch-project/opensearch-plugins/blob/main/CONVENTIONS.md#opensearch-dashboard-plugins . No issue with dashboardsMaps, as long as it doesn't affect the plugin building process. Can you verify once?